README.md

```diff
@@ -194,15 +182,5 @@ Use 'make' without arguments to see a list of available commands.
 
 ### Adding a new sub-package
 
-stactools is happy to take contributions of new subpackages for working with specific data types! Below is a list of steps to add a new subpackage:
-
-- Add the new subpackage as `stactools_{pkg}`, where `{pkg}` is a short name for the dataset the subpackage works with (e.g. "landsat")
-- Create a `setup.py`, `requirements.txt`, and `README.md` in the subpackage directory.
-- The code should exist in the `stactools/{pkg}/` directory in that package subdirectory. Note that the `stactools` does not have an __init__.py (look at the other subpackages for examples).
-- Add the subpackage to the appropriate variables in `scripts/env`
-- Add the subpackage requirements install section to
-`docker/Dockerfile`
-- Add the subpackage to the appropriate tables in the README.
-- Add documentation for the subpackage.
-- Add subpackage to .readthedocs.yml install
-- Add the subpackage to the `subpackages` list in the top-level `setup.py`, with `is_extra=True`. This will allow for an install of that specific subpackage with the pip extras syntax, e.g. `pip install stactools[aster]`.
+See the [`stactools` dataset subpackage template](https://github.com/stactools/stactools-dataset-template) for more information.
```

Comment:
change url
---

see the [`stactools` dataset subpackage template](https://github.com/stactools-packages/template) for more information.